packages/snap/src/index.ts

```diff
@@ -131,6 +134,75 @@ export const processRpcRequest: SnapRpcProcessor = async (
       return proof;
     }
 
+    case RpcMethods.GenTwitterFollowersCountProof: {
```

Comment:
I think we do not need a new method for the Twitter certificates. The snap should be general enough to handle different kinds of ZkCerts. This is important for devs outside of our team because they can not modify the snap.

Let me finish one commit putting those two methods together into one general one.

Reply:
Generalized it with these commits. Do you think this is ok?

- https://github.com/Galactica-corp/galactica-monorepo/pull/49/commits/8fae6c194644e20b36ac65b40ba7af54ff37da41
- https://github.com/Galactica-corp/galactica-monorepo/pull/49/commits/ccbc8c0725f6ce8dddcd838306e3eb77e4814f86
- https://github.com/Galactica-corp/galactica-monorepo/pull/49/commits/926e4ff5f2c98539308e66213d58b44177cdd523

Reply:
thanks, didn't think of that